pages/search.js

```diff
@@ -74,7 +74,7 @@ export default function Search(props) {
               display: flex;
             }
             .form_search {
-              height: 62vh;
+              height: 65vh;
```

Comment:
65vh adds a scroll bar in my window, but i don't know if it does that for anyone else